src/mbgl/tile/custom_geometry_tile.hpp

```diff
@@ -25,10 +33,12 @@ class CustomGeometryTile: public GeometryTile {
         const SourceQueryOptions&) override;
 
 private:
+    bool stale = true;
```

Comment:
`stale`: A flag indicating whether this tile was invalidated from an external call, and needs to re-fetch tile data.